src/MonetizationInterface.php

```diff
@@ -152,6 +153,23 @@ public function getSupportedCurrencies(): ?array;
    */
   public function getPrepaidBalanceReport(string $developer_id, \DateTimeImmutable $date, string $currency): ?string;
 
+  /**
+   * Returns a CSV string for revenue.
+   *
+   * @param string $developer_id
+   *   The developer id.
+   * @param \DateTimeImmutable $from_date
+   *   The from month for the report.
+   * @param \DateTimeImmutable $to_date
+   *   The to month for the report.
+   * @param string $currency
+   *   The currency id. Example: usd.
+   *
+   * @return null|string
+   *   A CSV string of revenue report.
+   */
+  public function getRevenueReport(string $developer_id, \DateTimeImmutable $from_date, \DateTimeImmutable $to_date, string $currency): ?string;
```

Comment:
We can't add new methods without breaking BC, although I totally feel this should go in the Monetization service class. This is something that we should discuss further with @cnovak , as the monetization service isn't really defining a contract, but it's more of a collection of utility functions, and I feel it'd be too much to release a new major version everytime we want to add to it.

Reply:
Yeah I've been wondering about this too. The `MonetizationInterface` is just a bunch of methods right now, might need to break this into actual interfaces.

It looks like we might need to start looking to 2.x branch soon.

Reply:
@arlina-espinoza I removed the new method from the interface and move it to the form for now. We can refactor this for `2.x`. Thank you.